data/saas/config/stripe_config.yml

```diff
@@ -241,12 +241,10 @@ saas_config:
             configuration:
               cursor_param: starting_after
               field: id
-        update:
-          method: POST
+        delete:
+          method: DELETE
+          ignore_errors: [404] # ignore errors if the card no longer exists
```

Comment:
included ignoring a 404 here, we ignore others already and this helps resolve errors for cards not found when a user approves two erasure requests simultaneously @galvana